pkg/config/config.go

```diff
@@ -126,6 +126,17 @@ func NewLocalConfigInfo(cfgDir string, errorNoLocalConfig bool) (*LocalConfigInf
 	return &c, nil
 }
 
+// LocalConfigFileExists checks if the local config file exists or not. an error while getting the
+// file name is also considered as file not existing
+func LocalConfigFileExists(cfgDir string) bool {
```

Comment:
I am refactoring some code in my https://github.com/openshift/odo/pull/1446 PR

We may have to move this code into utils as I have a feeling that this should be applied to most, if not all commands before execution. Checking to see if the config file exists.

Reply:
There is a flag in the `NewLocalConfigInfo` which will give u an error in case file doesn't exist which can be used everywhere else. I can maybe think of a way to reuse that.